.github/workflows/rnmobile-android-runner.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         native-test-name: [
-          gutenberg-editor-gallery
+          gutenberg-editor-blocks
```

Comment:
What do you think about renaming this to `gutenberg-editor-initial-html` and also keeping `gutenberg-editor-gallery`, but maybe making them run one after a another instead of running in parallel?

Reply:
I agree with the renaming. What is the rationale for keeping the gallery test ? I thought this test would cover all cases and I'm wondering if that test could be more flaky than this one. Also I'm wondering about the collective time of both them running one after the other as well. Let me know what you think.

Reply:
To be honest, I'm not interested in gallery block in this case 😄 But the gallery test checks some other aspects of the editor that are not covered by the initial HTML tests, mainly around the inserter menu, like opening the inserter menu, scrolling the inserter menu, and inserting a block.

The primary time-consuming task is building the js bundle and the native code (the ipa and the apk). The gallery test itself runs in ~5 seconds on iOS and ~50 seconds on Android. Android is slower due to iOS having a larger inserter, so there's no need to scroll the inserter to find the gallery block on iOS. 

Gallery test has been around for quite some time now, and I think it should be stable enough. But I realized now that we had made some changes around scrolling in this PR, that could be affecting the gallery test's stability. So it could be better to leave the gallery test out for now and maybe add it back in, in a separate PR making sure again it's flawless 💯